editors/code/package.json

```diff
@@ -120,6 +120,16 @@
                 "title": "Run",
                 "category": "Rust Analyzer"
             },
+            {
+                "command": "rust-analyzer.debug",
+                "title": "Debug",
+                "category": "Rust Analyzer"
+            },
+            {
+                "command": "rust-analyzer.newDebugConfig",
+                "title": "Generate launch configuration",
+                "category": "Rust Analyzer"
+            },
```

Comment:
Can we add a "save configuration" button to the quick pick? Looks like it supports some kind of extra buttons, tough it's not immediatelly clear how to use this funcitonality: https://code.visualstudio.com/api/references/vscode-api#QuickInputButton

Reply:
Thanks, I didn't know about this API. Yes, we can. Though the button is not very noticeable:
![save_cfg_button](https://user-images.githubusercontent.com/62505555/81860548-96ac3c80-956f-11ea-89d8-93937125eb19.png)

I will add the button for both run and debug quick picks, but would prefer to leave the command too. At least to show the possibility in the commands list.

Reply:
Seems reasonable!

While we are at it, maybe we should add `(from tasks.json)` to the label if we are loading persisted command

Reply:
The quick pick does not use any information from tasks.json, only from LSP. And I think we should leave it as it is. Or I did not get your idea.